### DISCUSSION TITLE
RLC-811A - no detection, no siren

### DISCUSSION THREAD

**Discussion by PetrStary** [fwestenberg/reolink_dev#385] [Q&A]
Hi there,
I am super new to this. I am trying to set-up my 811A camera.
For almost a month I am trying to fix the problem, that the Home Assistant never detects any motion correctly.

I have working live and auto view on the dashboard. IR light and Spotlight swithces are woking - Siren is not.

v0.42

RLC-811A
build 21110208
Config version: v3.1.0.0
Firmware: v3.1.0.688_21110208

I went out in front of the camera with the phone - reolink app sends notification correctly. HA does not know anything.
Can anyone help me please? Considering I am noob to HA. 

Thank you.

![image](https://user-images.githubusercontent.com/94073773/144626512-d32fa159-8704-45e6-bc93-c4fdfac80edb.png)

Last Event 5 hours ago - that is a non-sense.
But as you can see, standing in front of the camera - without any detection.

![image](https://user-images.githubusercontent.com/94073773/144631726-8e3f65ef-7e3a-48fa-b472-23e78f16775f.png)

**Comment by cpainchaud** [comment]
from https://github.com/fwestenberg/reolink_dev/blob/master/TSHOOT.md
Try this:
- If you are using a NVR, you must first enable ONVIF protocol via a menu which is only available via a HDMI screen connected to the NVR, without this then HA cannot be aware of a motion detection event.
- HomeAssistant must have an internal URL configured and: IT MUST NOT USE HTTPS : Reolink doesn't support HTTPS based Webhooks.
URL also should not be using a DNS name but an ip address instead unless you have a solid DNS setup your camera is probably not able to resolve your address

**Reply by PetrStary** [reply]
Thanks fo the reply.
* I am not using NVR.
* I dont think I am using https. It says unsecure and also the link is "http://home...."
![image](https://user-images.githubusercontent.com/94073773/144667603-7792974e-5053-48f9-a2be-d17b9a871ce9.png)
I made the IP address static in both home assistant and router settings. So should I open HA as http://192.168.12.110:8123/ instead of http://homeassistant.local:8123/ ? I am also using mobile app... 

Thank you for your help

**Reply by cpainchaud** [reply]
internal url here http://your_ip:8123/config/core  should be setup as http://your_ip:8123

**Reply by PetrStary** [reply]
It already says: (sorry for the language, but I am sure you know what it means)
![image](https://user-images.githubusercontent.com/94073773/144744701-c2975925-b78a-4723-89e7-3c11aae68dba.png)
So no change there was needed.
I better switched also Network like this:
![image](https://user-images.githubusercontent.com/94073773/144744751-99163dd5-d56e-43f2-8f38-8cb569778dba.png)

**Reply by paolomorabito** [reply]
Did you fix this issue? I have a RLC 811A, same problem. Motion detection is delayed or not recognized.
